samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor

```diff
@@ -307,8 +308,11 @@
             CommonHelper.Spinner = string.Empty;
             CommonHelper.CheckErrorOrEmpty<ListNode>(PagedNodeList, ref _tableView, ref _tableEmpty);
             StateHasChanged();
-            _publishEvent = await PublisherServiceEvents.NodePublishSubscribeByEndpointAsync(EndpointId,
+            if (PublishedNodes.Results.Count > 0)
```

Comment:
Is it possible that the count is > 0 on first render?  And if so, shouldn't you test for _publishEvent == null?

Reply:
the count can be > 0 at the first render. If you have an endpoint with previously published nodes, when you enter on this page the BrowseTreeAsync gives you the list of published nodes.
_publishevent on first render is always null, and if if there are already published nodes is initialized on first render, otherwise when you publish a node